src/test/java/org/folio/logintest/CookieParserTest.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.logintest;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThrows;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import org.folio.rest.impl.LoginAPI;
+import org.folio.util.TokenCookieParser;
+
+public class CookieParserTest {
```

Comment:
The class under test is `TokenCookieParser` so I expect the test name to be `TokenCookieParserTest`.

Reply:
Done